dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/communityManager/SingleDataRequestsTest.kt

```diff
@@ -43,6 +45,8 @@ class SingleDataRequestsTest {
     val apiAccessor = ApiAccessor()
     val jwtHelper = JwtAuthenticationHelper()
     private val requestControllerApi = RequestControllerApi(BASE_PATH_TO_COMMUNITY_MANAGER)
+    private val maxRequestsForUser = 10
+    private val dataReaderUserId = UUID.fromString("18b67ecc-1176-4506-8414-1e81661017ca")
```

Comment:
Why do we hard code the userId here and do not use TechnicalUser.Reader.technicalUserId as in the other cases in this PR?

Reply:
done

Reply:
Auf jeden Fall 👍 

gut gesehen

Habe es geändert